src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,65 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int INDEX_OF_NUMBER = 1;
+    private static final int SUPPLY_POSITION = 0;
+    private static final int BUY_POSITION = 1;
+    private static final int RESULT_POSITION = 2;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        WorkWithFile worker = new WorkWithFile();
+        worker.writeData(worker.createReport(worker.readFromFile(fromFileName)), toFileName);
```

Comment:
Make this entry more readable by adding variables for the results of the methods